File: Cogs/info.py
import discord
from discord.ext import commands
from datetime import *
from discord_components import *

class Info(commands.Cog):
    def __init__(self,Bot):
        self.Bot = Bot

    @commands.Cog.listener()
    async def on_ready(self):
        self.Bot.launch_time = datetime.utcnow()

    @commands.command()
    async def info(self,ctx):
        buttons = [
        [
            Button(style=ButtonStyle.URL, label='Support Server', url="https://discord.gg/tXd9gPtKCj"),
            Button(style=ButtonStyle.URL, label="Vote",url = "https://top.gg/bot/869500014899122246/vote")
        ]
    ]
        total_members = sum([len(guild.members) for guild in self.Bot.guilds])
        delta_uptime = datetime.utcnow() - self.Bot.launch_time
        hours, remainder = divmod(int(delta_uptime.total_seconds()), 3600)
        minutes, seconds = divmod(remainder, 60)
        days, hours = divmod(hours, 24)
        embed=discord.Embed(title="**Info**",description="Calculator is a discord bot developed to help you for your basic math operations.",colour=discord.Color.from_rgb(0,255,148))
        embed.set_footer(text=f"Requested By {ctx.message.author}",icon_url=ctx.message.author.avatar_url)
        embed.set_thumbnail(url=self.Bot.user.avatar_url)
        embed.add_field(name="**Author**",value="[451#2950](https://discordapp.com/users/453613270725558292)")
        embed.add_field(name="Library",value="[discord.py](https://discordpy.readthedocs.io/en/stable/)")
        embed.add_field(name="Uptime",value=f"{days}d, {hours}h, {minutes}m")
        embed.add_field(name="Servers",value=len(self.Bot.guilds))
        embed.add_field(name="Users",value=total_members)
        embed.add_field(name="Ping",value=f"{round(self.Bot.latency * 1000)}ms")
        embed.add_field(name="Links",value="[Github](https://github.com/451EK/Calculator) | [Invite](https://discord.com/oauth2/authorize?client_id=869500014899122246&scope=bot&permissions=201706561)",inline=False)
        embed.timestamp = datetime.utcnow()
        await ctx.send(components=buttons,embed=embed)

def setup(Bot):
    Bot.add_cog(Info(Bot))

File: Commands/Info/info.js
const Discord = require("discord.js")
const { MessageEmbed, MessageButton, MessageActionRow } = require("discord.js")
const ms = require("ms")

module.exports = {
    name : "info",
    usage: "`-info`",
    description: "Shows information about the bot.",
    run : async(client,message) => {
        const components = new Discord.MessageActionRow()
            .addComponents([
                new Discord.MessageButton()
                .setLabel("Support Server")
                .setURL("https://discord.gg/tXd9gPtKCj")
                .setStyle("LINK"),

                new Discord.MessageButton()
                .setLabel("Vote")
                .setURL("https://top.gg/bot/869500014899122246/vote")
                .setStyle("LINK"),
            ]);
        const ping = ~~client.ws.ping
        let allUsers = [];
        client.guilds.cache.forEach(guilds => allUsers.push(guilds.memberCount))
        let users = allUsers.reduce((a, b) => a + b)
        const servers = client.guilds.cache.size
        let totalSeconds = process.uptime();
        let d = Math.floor((totalSeconds % 31536000) / 86400); // days
        let h = Math.floor((totalSeconds / 3600) % 24); // hours
        let m = Math.floor((totalSeconds / 60) % 60); // minutes
        const embed = new MessageEmbed()
        .setTitle(title="**Info**")
        .setDescription(this.description="Calculator is a discord bot developed to help you for your basic math operations.")
        .setFields(
            {name : "Author",value: "[451#2950](https://discord.com/users/453613270725558292)", inline:true},
            {name: "Library", value:"[discord.js](https://discord.js.org/#/)",inline:true},
            {name: "Uptime", value:`${d}d, ${h}h, ${m}m`,inline:true},
            {name: "Users", value:`${users}`,inline:true},
            {name:"Servers",value:`${servers}`,inline:true},
            {name:"Ping", value:`${ping}ms`,inline:true},
            {name: "Links", value:"[Github](https://github.com/M451z/Calculator) | [Invite](https://discord.com/api/oauth2/authorize?client_id=869500014899122246&permissions=259845909568&scope=bot%20applications.commands)"}
        )
        .setThumbnail(url="https://i.imgur.com/AJXNGEW.png")
        .setTimestamp()
        .setFooter(text=`Requested by ${message.author.tag}`,iconURL = message.author.avatarURL())
        .setColor("#00ff94")
        message.channel.send({embeds:[embed],components:[components]})
    }
}